clang/lib/Frontend/FrontendActions.cpp

```diff
@@ -80,6 +80,22 @@ void ReadPCHAndPreprocessAction::ExecuteAction() {
   do {
     PP.Lex(Tok);
   } while (Tok.isNot(tok::eof));
+
+  // Import additional modules.
```

Comment:
Shouldn't this happen before lexing the TU? Clients may expect that symbols from the requested modules are visible in the TU itself.